tests/Identifier/Normalizer/DateTimeIdentifierNormalizerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Identifier\Normalizer;
+
+use ApiPlatform\Core\Identifier\Normalizer\DateTimeIdentifierDenormalizer;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class DateTimeIdentifierNormalizerTest extends TestCase
```

Comment:
add a `supportsDenormalization` test?

Reply:
Not necessary, the `supportsDenormalization` method is not overrode and the base one is already tested in Symfony.